roles/openshift_node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 os_firewall_allow:
-- service: OpenShift kubelet
+- service: Kubernetes kubelet
```

Comment:
@aveshagarwal The main issue with modifying iptables rules, is that there is no way to ensure concurrency between multiple processes modifying iptables (docker, kubernetes, openshift-sdn, openshift-ansible).  

The only synchronization primitives offered in the iptables librarires are considered private and the developers do not guarantee stability even between minor releases, so there really isn't a way to ensure that multiple actors do not step on each other.

I've limited the exposure to this as much as I could by using a separate Table, but it is still hacky at best.